cpu/cpu_solaris.go

```diff
@@ -160,9 +160,7 @@ func parseISAInfo(cmdOutput string) ([]string, error) {
 	}
 
 	flags := make([]string, len(words)-4)
-	for i, val := range words[4:] { //nolint:gosimple //FIXME
-		flags[i] = val
-	}
+	copy(flags, words[4:])
 	sort.Strings(flags)
```

Comment:
I don't understand the code was very complicated, but now it's clearer.

What about using this?

```go
         flags := words[4:]
         sort.Strings(flags)
```

Reply:
@ccoVeille Thank you for your suggestion. I agree with you. 
Could you please update the code? > @mmorel-35